workflow/gccontroller/gc_controller.go

```diff
@@ -215,7 +215,7 @@ func (c *Controller) deleteWorkflow(ctx context.Context, key string) error {
 			return err
 		}
 	} else {
-		log.Infof("Successfully deleted '%s'", key)
+		log.Infof("Successfully request '%s' to be deleted", key)
```

Comment:
This would depend on `WF_DEL_PROPAGATION_POLICY`

Reply:
@terrytangyuan Hi, thank you for comments. I think this issue is not related to k8s DeletionPropagation. The log `Successfully deleted` should depend on the real status of workflow, which is whether existed or deleted. In our case, our workflow has many different finalizers attached by different services, any existing finalizer will prevent the deletion, which will be confused by this log msg.